treeherder/config/settings.py

```diff
@@ -98,6 +98,11 @@
     'default': env.db_url('DATABASE_URL'),
 }
 
+# Only used when syncing local database with prototype/production
```

Comment:
I think it'd be worth emphasizing this will be querying the prod-replica, not the actual production database.

Reply:
> I think it'd be worth emphasizing this will be querying the prod-replica, not the actual production database.

Yes, I see your point. Will update.